Delicate_Flowers_Vignette-AR.qmd

```diff
@@ -0,0 +1,222 @@
+
+[^delicate_flowers_vignette-ar-2]: [Sarah Morgan Dawson, “A Confederate Girl’s Diary: 1842-1909,” (Documenting the South: UNC. Accessed February 12, 2024)](https://www.zotero.org/google-docs/?broken=MNVVUt), 24.
+
+### Word Vector Analysis
+
+Word vector analysis seemed like the obvious choice to distantly view this corpus. Each of these women had a unique writing style and their own preferred vocabularies. Additionally, Southern white women were accustomed to using euphemisms to discuss delicate topics, such as sex.[^delicate_flowers_vignette-ar-3] Word vectors promised to mitigate these complications.
+
+[^delicate_flowers_vignette-ar-3]: E. Susan Barber and Charles F. Ritter, “Dangerous Liaisons: Working Women and Sexual Justice in the American Civil War.” European journal of American studies 10, no. 1 (March 26, 2015).
+
+```{r}
+#| eval: false
+# Need to wrap these in an if statement so that if the file generated exists, it doesn't rerun the model, it just loads the model file (.txt or .bin respectively)
+prep_word2vec(origin="text files",destination="textfiles.txt",lowercase=T,bundle_ngrams=1)
+
+model = train_word2vec("textfiles.txt","textfiles.bin",vectors=400,threads=6,window=12,iter=3,negative_samples=5)
+```
+
+In order to limit my results to *women's* experiences, I fed the model feminine terms and told it reject masculine ones. The results were surprising. The first fourteen words that appear, most notably "moan," "tear," "weep," "hush," "sobs," "shrill," "lone," "sacrificed," and "screams," carry a disheartened connotation. This aligns with the tone of Margaret Ann Meta Morris Grimball’s diary, which I also read, but not Dawson's or Emma LeConte's. The fourteenth word to appear in this model, "patriotic," is what I expected to find. Why were Dawson and LeConte's voices suppressed with this method?
+
+```{r}
+wowords <- model[[c("female", "females", "women", "woman", "feminine", "she", "woman's")]] %>% reject(model[[c("male", "males", "men", "man", "masculine", "he", "men's")]])
+model %>% nearest_to(wowords, 100)
+
+```
+
+### Topic Modeling
+
+I ran into the same problem when I attempted to topic model the corpus. The results were logical but they did not align with what I expected to find – defiance, disdain for the Union, and courage.
+
+```{r}
+diary.lda <- LDA(diary.dtm, k = 15, control = list(seed = 12345), method = "Gibbs", alpha = .7)
+diary.lda
+
+diary.topics <- tidy(diary.lda, matrix = "beta")
+head(diary.topics)
+
+diary.top.terms <- diary.topics %>%
+  arrange(desc(beta)) %>% 
+  group_by(topic) %>% slice(1:8)
+
+
+diary.top.terms %>%
+  mutate(term = reorder_within(term, beta, topic)) %>%
+  ggplot(aes(beta, term, fill = factor(topic))) +
+  geom_col(show.legend = FALSE) +
+  facet_wrap(~ topic, scales = "free") +
+  scale_y_reordered()
```

Comment:
I'm curious what happens when you pull the gamma values here and try to graph topics over time. You could also look at which topics are most prevalent in which diaries.